src/MongoDB/ReadPreference.c

```diff
@@ -526,16 +477,12 @@ static PHP_METHOD(MongoDB_Driver_ReadPreference, serialize)
 	}
 
 	tags                = mongoc_read_prefs_get_tags(intern->read_preference);
-	mode                = mongoc_read_prefs_get_mode(intern->read_preference);
```

Comment:
Note: this will conflict with https://github.com/mongodb/mongo-php-driver/pull/1663, depending on which gets merged first.